libraries/core/src/main/java/com/paritytrading/philadelphia/FIXValue.java

```diff
@@ -85,6 +85,32 @@ public void set(FIXValue value) {
         System.arraycopy(value.bytes, 0, bytes, 0, length + 1);
     }
 
+    /**
+     * Get the value as a boolean.
+     *
+     * @return the value as a boolean
+     * @throws FIXValueFormatException if the value is not a boolean
+     */
+    public boolean asBoolean() {
+        if (length != 1)
```

Comment:
Is this a sufficient check that it's not a boolean? `notBoolean` and `notChar` are less meaningful than the other sanity checks (`notDigit` etc).